### ISSUE TITLE
Failed PoSt after v1.5.0 upgrade

### ISSUE THREAD

**Issue by funktafide** [filecoin-project/lotus#5677]
After v1.5.0 upgrade I am failing PoSt.  I have no active sealing jobs since the upgrade, so resources shouldn't be a problem.  It looks like it's missing something based on the error.  I grep'd the miner log for post and received the following error:

2021-02-24T23:34:54.277 INFO filcrypto::proofs::api > generate_winning_post: start
2021-02-24T23:34:54.435 INFO filcrypto::proofs::api > generate_winning_post: finish
2021-02-24T23:34:54.435+0800	ERROR	miner	miner/warmup.go:82	winning PoSt warmup failed	{"error": "failed to compute proof: Sealed replica does not exist\n\nStack backtrace:\n   0: filecoin_proofs::api::post::PrivateReplicaInfo<Tree>::new\n   1: filecoin_proofs_api::post::generate_winning_post_inner\n   2: filecoin_proofs_api::post::generate_winning_post\n   3: <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once\n   4: ffi_toolkit::catch_panic_response\n   5: fil_generate_winning_post\n   6: _cgo_a8fa62747d41_Cfunc_fil_generate_winning_post\n             at /tmp/go-build/cgo-gcc-prolog:640\n   7: runtime.asmcgocall\n             at /usr/local/go/src/runtime/asm_amd64.s:656", "errorVerbose": "failed to compute proof:\n    github.com/filecoin-project/lotus/miner.(*Miner).winPoStWarmup\n        /home/funk/lotus/miner/warmup.go:72\n  - Sealed replica does not exist\n    \n    Stack backtrace:\n       0: filecoin_proofs::api::post::PrivateReplicaInfo<Tree>::new\n       1: filecoin_proofs_api::post::generate_winning_post_inner\n       2: filecoin_proofs_api::post::generate_winning_post\n       3: <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once\n       4: ffi_toolkit::catch_panic_response\n       5: fil_generate_winning_post\n       6: _cgo_a8fa62747d41_Cfunc_fil_generate_winning_post\n                 at /tmp/go-build/cgo-gcc-prolog:640\n       7: runtime.asmcgocall\n                 at /usr/local/go/src/runtime/asm_amd64.s:656\n    github.com/filecoin-project/filecoin-ffi.GenerateWinningPoSt\n    \t/home/funk/lotus/extern/filecoin-ffi/proofs.go:545\n    github.com/filecoin-project/lotus/extern/sector-storage/ffiwrapper.(*Sealer).GenerateWinningPoSt\n    \t/home/funk/lotus/extern/sector-storage/ffiwrapper/verifier_cgo.go:30\n    github.com/filecoin-project/lotus/storage.(*StorageWpp).ComputeProof\n    \t/home/funk/lotus/storage/miner.go:259\n    github.com/filecoin-project/lotus/miner.(*Miner).winPoStWarmup\n    \t/home/funk/lotus/miner/warmup.go:64\n    github.com/filecoin-project/lotus/miner.(*Miner).doWinPoStWarmup\n    \t/home/funk/lotus/miner/warmup.go:80\n    runtime.goexit\n    \t/usr/local/go/src/runtime/asm_amd64.s:1374"}
2021-02-25T00:08:30.297+0800	WARN	storageminer	storage/wdpost_run.go:226	Checked sectors	{"checked": 460, "good": 0}
2021-02-25T01:08:30.281+0800	WARN	storageminer	storage/wdpost_run.go:226	Checked sectors	{"checked": 9, "good": 0}

Lotus Version:
Daemon:  1.5.0+mainnet+git.55b85f314+api1.1.0
Local: lotus version 1.5.0+mainnet+git.55b85f314

**Comment by jennijuju** [comment]
`failed to compute proof: Sealed replica does not exist\` seems like you removed some essential datastore.

**Comment by funktafide** [comment]
I just did the normal upgrade-in-place steps to go from v1.4.2 up to v1.5.0.  No datastores were moved, deleted or modified.  Maybe something just went wrong on my system during the upgrade.

I talked to stuberman earlier and I'm going to wipe out my lotus directory and rebuild. I will let it run tonight without sealing or anything to see how PoSt goes.

**Comment by magik6k** [comment]
Can you try running `lotus-miner proving check --slow [deadline]` for failed deadlines?

**Comment by funktafide** [comment]
@magik6k All sectors returned as good after running.

Since my initial post, I have removed the Lotus folder and "re-installed" v1.5.0.  During the install I was receiving 
bash: go: command not found
expr: syntax error: unexpected argument '1015005'
when running sudo make install.  Since a lot of my errors were go related in the log, I went through to ensure the go PATH variable was applying under sudo as well as my user account. Once that was worked out I was able to complete the install without errors.

I also cleared my chain data and re-syncd from the most current snapshot.  Things look good so far.  I will update tomorrow once I go through post later tonight.
